packages/x-charts-pro/src/ChartOverview/ChartOverview.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { useXAxes, useYAxes } from '@mui/x-charts/hooks';
+import { ChartAxisOverview } from './ChartAxisOverview';
+
+/**
+ * Renders the overview for all x and y axes that have an overview enabled.
+ */
+export function ChartOverview() {
+  const { xAxisIds, xAxis: xAxes } = useXAxes();
+  const { yAxisIds, yAxis: yAxes } = useYAxes();
+
+  return (
+    <React.Fragment>
+      {xAxisIds.map((axisId) => {
+        const xAxis = xAxes[axisId];
+
+        const overview = typeof xAxis?.zoom === 'object' ? xAxis.zoom.overview : undefined;
```

Comment:
Would probably make sense to defaultize the values to a full zoom config instead of having to check if zoom is object everytime 🤔